hi-ml-cpath/src/health_cpath/utils/plots_utils.py

```diff
@@ -87,11 +87,15 @@ def save_top_and_bottom_tiles(
     top_tiles_fig = plot_attention_tiles(
         slide_node=slide_node, case=case, top=True, num_columns=num_columns, figsize=figsize
     )
+    if top_tiles_fig is None:
```

Comment:
I would suggest moving this check to save_figure function? to avoid duplicating code? 
make fig optional and only save when it's not None?